cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/OperationInspectInfo.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.dataRecovery;
+import io.pravega.segmentstore.server.logs.operations.Operation;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class OperationInspectInfo extends Operation {
+    public static final long DEFAULT_ABSENT_VALUE = Long.MIN_VALUE + 1;
+    private final String operationTypeString;
+    private final long length;
+    private final long segmentId;
+    private final long offset;
+    private final long attributes;
+
+    public OperationInspectInfo(long sequenceNumber, String operationTypeString, long length, long segmentId, long offset, long attributes) {
+        super();
+        setSequenceNumber(sequenceNumber);
+        this.operationTypeString = operationTypeString;
+        this.length = length;
+        this.segmentId = segmentId;
+        this.offset = offset;
+        this.attributes = attributes;
+    }
+
+    @Override
+    public String toString() {
+        return "{ " + operationTypeString +
+                ", SequenceNumber=" + this.getSequenceNumber() +
```

Comment:
Can you initialize the right values in the constructor and here just print the actual data that the object contains?

Reply:
done